web/source/dom/domEventHandlers.ts

```diff
@@ -774,8 +780,12 @@ namespace com.keyman.dom {
      * Handle the touch move event for an input element
      */         
     dragInput: (e: TouchEvent|MouseEvent) => void = function(this: DOMTouchHandlers, e: TouchEvent|MouseEvent) {
-      // Prevent dragging window 
-      e.preventDefault();
+      // Prevent dragging window
+      if(e.cancelable) {
```

Comment:
I assume this fix is unrelated to the primary fix and diagnostic log?

Reply:
This was generating extraneous error messages that we don't need to capture.  It mostly shows up during development testing, at least in my experience.

When it does show, it gets _very_ noisy.  The following is taken from (what a user would perceive as) a simple, single scroll.  (Took several attempts to repro, but when it _does_ trigger...)

<img width="736" alt="Screen Shot 2021-04-19 at 8 29 44 AM" src="https://user-images.githubusercontent.com/25213402/115169883-7bd03980-a0e9-11eb-9acc-3cb3784b889e.png">

Note that Chrome's error text tells me rather directly to make this adjustment.